MetaMask/metamask-extension
# Update sentry to v7.53.0

## Explanation

This updates the sentry SDK packages we use.

This update in versions does come with breaking changes, but as far as I can tell, they don't affect our usage: https://github.com/getsentry/sentry-javascript/releases/tag/7.0.0

## Manual Testing Steps

Build the PR branch locally following the instructions in the Sentry section of https://github.com/MetaMask/metamask-extension/tree/develop/development#debugging-in-sentry

You should be able to see errors from the build of this branch on the sentry instance you created to follow those steps

## Pre-merge author checklist


## Pre-merge reviewer checklist


If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.
